node/examples/electron/object_detection/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "object_detection",
+  "version": "0.0.1",
+  "description": "An Electon.js example of object_detection reusing webnn-native",
```

Comment:
object_detection reusing => Object Detection using

Reply:
Done

Reply:
Do you miss this modification?